src/foam/graphics/CView.js

```diff
@@ -468,7 +468,6 @@ foam.CLASS({
       name: 'border'
     },
     {
-      class: 'Color',
```

Comment:
Why removing this?

Reply:
So **Cview.color** is used to set both `x.fillStyle` and `x.strokeStyle`, both **fillStyle** and **strokeStyle** were made to accept both: 
1. A string parsed as a color value
2. CanvasGradient object
3. CanvasPattern object

taken from MDN here: 
https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/fillStyle
https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/strokeStyle

So I removed the propertyType on cview.color so that a user could also set a `foam.graphics.Gradient` object onto cview.color as well. This way we can set a gradient inside fillStyle and strokeStyle.

If we wanted to separate this out then I would propose we spec both a fillStyle and strokeStyle property on cview instead of color. Under this solution **fillStyle** and **strokeStyle** would have no propertyType so that we could pass in either a String color or a gradient object.